app/models/journeys.rb

```diff
@@ -0,0 +1,48 @@
+# Policies should not control the front-end journey as one journey can be an application
+# for more than one policy
+class Journeys
+  ALL = [
+    {
+      routing_name: "student-loans",
```

Comment:
I think there's probably a way we can move this routing name out of the model and into the routes.rb where it belongs (possibly by mounting or with a module), but this is a good step.

Reply:
Agreed, sensible place to put it for now.